lib/transport_web/api/controllers/dataset_controller.ex

```diff
@@ -6,11 +6,11 @@ defmodule TransportWeb.API.DatasetController do
     render(conn, data: ReusableData.list_datasets)
   end
 
-  def show(%Plug.Conn{} = conn, %{"slug" => slug}) do
-    render(conn, data: ReusableData.get_dataset(slug))
+  def show(%Plug.Conn{} = conn, %{"id" => id}) do
```

Comment:
We want to use `id` while communicating internally with data.gouv.fr. But do we want to do it in our front-end ?

Reply:
I’d be in favor to use ids everywhere, excepted in what is shown to the user. That would be only the url.

Hence, here, I would keep the id, but adapt it bellow